HACKING.md

```diff
@@ -454,10 +477,10 @@ Ensure both node (current LTS) and yarn are installed.
 From the root of the MAAS UI project run:
 
 ```shell
-dotrun build-all
+yarn build-all
 ```
```

Comment:
A small drive-by here, the copy was no longer accurate since our migration to single-spa. Anyone wanting to build `maas-ui` is also not likely to use `dotrun` outside of the team.